src/ring_theory/finiteness.lean

```diff
@@ -239,6 +239,18 @@ begin
     (mv_polynomial.pempty_alg_equiv R)
 end
 
+/-- The quotient of a finitely presented algebra by a finitely generated ideal is finitely
+presented. -/
+lemma quotient {I : ideal A} (h : submodule.fg I) (hfp : finitely_presented R A) :
```

Comment:
Could you please add the variant for surjective maps with `fg` kernel?

Reply:
No problem. I can do it when #6166 is merged, so the proof will be just one line.

Reply:
Ok, I'll add #6166 as dep to this PR